packages/main/src/RPA/JavaAccessBridge.py

```diff
@@ -103,7 +98,6 @@ def __init__(
             self.showing = "showing" in self.states
             self.focusable = "focusable" in self.states
             self.node = node
-            self.internal = internal_node
```

Comment:
This isn't used nor important and creates confusion. (we already have our passed `.node` for that purpose)